src/quo2/components/account_avatar.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.account-avatar
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(def light-mode-color
+  {:dark colors/purple-opa-60
+   :light colors/purple-opa-20})
+
+(def sizes
+  {:xxl 80
+   :xl 48
+   :large 32
+   :medium 24
+   :small 20})
+
+(def border-radii
+  {:xxl 16
+   :xl 12
+   :large 10
+   :medium 8
+   :small 6})
+
+(def inner-icon-sizes
+  {:xxl 48
+   :xl 24
+   :large 20
+   :medium 12
+   :small 12})
+
+(defn account-avatar
+  [{:keys [size dark? icon]}]
+  (let [icon-color (if dark?
+                     (:dark light-mode-color)
+                     (:light light-mode-color))
+        avatar-size (size sizes)
+        avatar-border-radius (size border-radii)
+        inner-icon-size (size inner-icon-sizes)]
+    [rn/view {:style {:width avatar-size
+                      :background-color icon-color
+                      :height avatar-size
+                      :border-radius avatar-border-radius
+                      :justify-content :center
+                      :align-items :center}}
+     [icons/icon (keyword (str "main-icons/" icon))
```

Comment:
could you elaborate on why icon is a string? 
also `main-icons` ns is used only for IDE, it isn't used in icon component,

Reply:
Hey @flexsurfer
I wanted to create a namespaced keyword to make the icon dynamic

Reply:
What do you mean by main-icons is only for IDE?

Reply:
i mean its just for find function in IDE, when you want to find keyword, to find icon keyword this ns is used, but in icon component we remove this ns,

Reply:
> 

what do you mean ?

Reply:
Oh i see now, but the usages I see everywhere has this form

`:main-icons/wallet`

And namespaced keywords help show ownership of a keyword by namespacing it


If we have 2 id keywords they would shadow each other on usage

`:user/id`
`product/id`

But with namespaced keywords, each is tied to a certain namespace

Reply:
> i mean its just for find function in IDE, when you want to find keyword, to find icon keyword this ns is used, but in icon component we remove this ns,

So you mean i should do it like this
`[Icons/icon :peach16 {props}]` ?

Reply:
yeah i mean it should be keyword not string `[icons/icon icon]`

Reply:
Oh I already converted the string to a keyword here using the `keyword` function 
I used `str` to make use of the string coming from the text field and form a namespaced keyword of
`:main-icons/<DYNAMIC-ICON>`

Reply:
if `icon` is already a keyword, then all is good just use `[icons/icon icon]`

Reply:
Yeah I'd do that
But it isn't, it's passed as a string prop

Reply:
it should be a keyword then

Reply:
If we passed it as a keyword, We'd have to make a list of keywords for all possible emojis .. Which one would you prefer ? converting the string to a keyword or make a keyword only list of emojis ?

Reply:
keyword only list of emojis

Reply:
Done